.github/workflows/docker-cli.yaml

```diff
@@ -0,0 +1,28 @@
+name: Build Docker Image
+
+on:
+  workflow_dispatch:
+    inputs:
+      COMPOSEDB_CLI_VERSION:
+        description: 'ComposeDB CLI version'
+        default: 'latest'
```

Comment:
Can we detect the CLI version and use that? I feel like the `latest` tag should apply to all `main` builds and the actual CLI version tag should apply when a new release is created.

Reply:
Updated to take a branch and version from that branch.

Unsure how the releases are created.